src/app/modules/details/test-exec-details.module.ts

```diff
@@ -0,0 +1,16 @@
+import { NgModule } from '@angular/core';
+import { CommonModule } from '@angular/common';
+import { TestExecDetailsComponent } from './test-exec-details.component';
+
+@NgModule({
+  imports: [
+    CommonModule
```

Comment:
given that we use the MessagingService you could also add "MessagingService.forRoot()", right?

Reply:
right, but I put it into `AppModule` instead, analogous to test-exec-navigator.